target/linux/generic/hack-6.12/910-kobject_uevent.patch

```diff
@@ -20,7 +29,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +	u64 seq;
 +
 +	mutex_lock(&uevent_sock_mutex);
-+	seq = ++uevent_seqnum;
++	seq = atomic64_inc_return(&uevent_seqnum);
 +	mutex_unlock(&uevent_sock_mutex);
```

Comment:
We use atomic here. Drop the mutex

Reply:
You mean to leave only:
```
{
	return atomic64_inc_return(&uevent_seqnum);
}
```
?

Reply:
yes

Reply:
Done in local PR https://github.com/namiltd/openwrt/pull/35